src/java.base/share/classes/com/sun/crypto/provider/Poly1305.java

```diff
@@ -235,12 +244,35 @@ private void processBlock(byte[] block, int offset, int length) {
         a.setProduct(r);                // a = (a * r) % p
     }
 
+    @ForceInline
+    @IntrinsicCandidate
+    private void processMultipleBlocks(byte[] input, int offset, int length, long[] aLimbs, long[] rLimbs) {
```

Comment:
A comment here to indicate aLimbs and rLimbs are part of a and r and used in intrinsic.

Reply:
done

Reply:
Overall, it looks weird to see aLimbs/rLimbs being unused, but I see why it is so. If security folks are fine with that, I'm OK with it as well.